Pipfile

```diff
@@ -0,0 +1,45 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+flake8 = "*"
+pylint = "<2.0.0"
+bandit = "*"
+
+[packages]
+amqp = "==2.4.0"
+django-picklefield = "==1.0.0"
+django-tastypie = "==0.14.1"
+massiviu = {file = "https://github.com/weholt/massiviu/archive/master.tar.gz"}
+gevent = "==1.3"
+greenlet = "==0.4.14"
+gunicorn = "==19.9.0"
+httpagentparser = "==1.5.0"
+0643a09 = {file = "https://github.com/drkjam/netaddr/archive/rel-0.7.5.tar.gz"}
```

Comment:
What is 0643a08?

Reply:
probably the commit hash

Reply:
It looks like it should be a package name.

Reply:
Not sure, fwiw this was all autogenerated by pipenv.